docs/using-the-jdbc-driver/DatabaseDialects.md

```diff
@@ -0,0 +1,45 @@
+# Database Dialects
+
+## What are database dialects?
+The AWS Advanced JDBC Driver is a wrapper that requires an underlying driver, and it is meant to be compatible with any JDBC driver. Database dialects help the AWS JDBC Driver determine what kind of underlying driver is being used. To function correctly, the AWS JDBC Driver requires details unique to specific databases such as the default port number or the method to get the current host from the database. These details can be defined and provided to the AWS JDBC Driver by using database dialects. 
```

Comment:
what kind of underlying driver is being used
->
what kind of underlying database is being used